src/test/resources/fixtures/item.csv

```diff
@@ -0,0 +1,2 @@
+text,fields,register
+"The Entry 1",field1,value1

```

Comment:
oops, just saw `writeEntriesTo_writesLists` above already covers this case. never mind, move along... 😄